elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -111,17 +111,23 @@ class PfeContentSet extends PFElement {
     return this.parentNode ? this.parentNode.offsetWidth > breakpointValue : window.outerWidth > breakpointValue;
   }
 
+  get renderedComponent() {
+    return this.querySelector(`[slot="rendered-component"]`);
+  }
+
   get tabs() {
-    return this.shadowRoot.querySelector(`${PfeTabs.tag}[visible-at="large"]`);
+    return this.querySelector(`pfe-tabs[slot="rendered-component"]`);
   }
 
   get accordion() {
-    return this.shadowRoot.querySelector(`${PfeAccordion.tag}[visible-at="small"]`);
+    return this.querySelector(`pfe-accordion[slot="rendered-component"]`);
   }
 
   constructor() {
     super(PfeContentSet);
 
+    this.fetchComponent = this.fetchComponent.bind(this);
+
     this._mutationHandler = this._mutationHandler.bind(this);
```

Comment:
Can we watch the slot with the mutation observer instead?  Mutation observer on the component is picking up when tabs & accordion update.